src/release/breaking-changes/boxdecoration-shape-migration.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migration guide for BoxDecoration.circle
+description: Removal of BoxDecoration.circle and how to migrate
+---
+
+## Summary
+
+`BoxDecoration(shape: BoxShape.circle)` was removed, and existing apps should
+use `ShapeDecoration(shape: CircleBorder())` instead.
+
+## Context
+
+According to [`Hixie comment`]):
+> Each enum value is a lot of new code in a bunch of places, including in code
+> we don't control (e.g. app code, that's why the Google tests are failing),
+> so we'd need to write a migration guide and migrate a bunch of code.
+> Each new enum value requires new code in a bunch of places.
+> That new code likely won't get tree-shaken out, since I don't think the
+> compiler can know which values are never used. So this has a permanent
+> binary size cost even for people who don't use it.
+> In principle, we've moved to ShapeBorders rather than BoxShape for new
+> shapes, and the existence of BoxShape is just a legacy feature at this
+> point. Adding more shapes encourages people to use BoxShape in the future.
+
+Due to this, BoxShape isn't needed anymore and can be a source of confusion.
+ShapeDecoration also aligns better to the rest of the Flutter API.
```

Comment:
**suggestion:** Make the copy more concise.
```suggestion
Use `ShapeDecoration`. It aligns with the rest of the Flutter API. `BoxShape` has been deprecated.
Continuing to use that feature can cause confusion.
```

Reply:
The @Hixie comment is being removed from this PR, right? 👍

Reply:
Yes.